### ISSUE TITLE
Tiny Data

### ISSUE THREAD

**Issue by jckantor** [ndcbe/data-and-computing#7]
This project is taking shape and will be a huge addition for the curriculum.  In reviewing the contents, one suggestion would be to include notes on organizing data for subsequent analysis.  Students often spend a lot of time using spreadsheets and other tools organizing and formatting data into complex tables that make subsequent imports and analysis difficult or impossible. 

I wrote up some notes on "Tidy Data", a well documented and commonly used set of principles for organizing data for analysis. There's a strong theoretical foundation (third normal form for relational data sets) of why this works so well, but all users need to know are the basic principles. This streamlines subsequent analysis whether using Python, R, spreadsheets, or any other system for data analysis.  This might fit nicely in the chapter "Descriptive Statistics and Visualization".  Feel free to cut and paste as much of these as you might find useful.

https://jckantor.github.io/cbe30338-book/python/A.05-Tidy-Data-and-Pandas.html

**Comment by adowling2** [comment]
This is a great suggestion. At a minimum, I am going to send the CBE 60258 students to this link for supplemental materials and reading. In CBE 20258, we provide most of the data. (The assignments are canned.) But we should strategize about how to integrate Tiny Data into the junior lab.

**Comment by jckantor** [comment]
Exactly.  These notes were prepared for the Junior lab experiment I'm teaching (heat exchanger), but put into the control notes for lack of a repo for Junior lab.
